apps/admin-server/src/hooks/use-status.tsx

```diff
@@ -0,0 +1,21 @@
+import useSWR from 'swr';
+
+export default function useStatuses(projectId?: string, id?: string) {
```

Comment:
Kan je hier een expliciete export van maken? 
Dus export function useStatus?

Reply:
Eh, met alle plezier, maar dat zou dan overal zo moeten werken: all die hooks zien er zo uit. En dan is consistentie belangrijker dan correctheid.

Misschien een Trello ticket?

Ik heb wel het meervoud gwijzigd naar enkelvoud. Ook in use-tag.

Reply:
Trello ticket kan wel